lib/hardware_interface/fcb_messages/fcb_message_definitions.dart

```diff
@@ -57,10 +57,19 @@ class AltitudeInfoMessage extends BaseMessage {
         ]);
 }
 
+List<bool> parsePyroContinuity(int pyroDec) {
+  List<bool> pyroContinuityList = [];
+  // Convert the integer pyroDec to an array of booleans that represent Pyro Continuity for each channel
+  for (var i = 0; i < 8; i++) {
```

Comment:
Re: previous comment, just wanted to check if it was considered making this 8 come from the packet definition. 

It's not so much an issue here but in the future I think it'll be valuable to be able to to know the sizes of packets/packet members dynamically from the definitions. However, not a big deal if it's really difficult to access that information in a nice way.

Reply:
offline: not easy